lib/message.rb

```diff
@@ -0,0 +1,13 @@
+require './lib/morse_translator'
+class Message
+  attr_reader :text,
+              :translator
+  def initialize(text,translator)
+    @text = text
+    @translator = translator
+  end
+
+  def translation
+    translator.translate(text)
```

Comment:
When you are inside the class, we generally want to see you use the instance variable @translator, rather than the method #translator